molgenis-python/src/main/java/org/molgenis/python/PythonScriptRunner.java

```diff
@@ -31,8 +32,15 @@ public boolean hasFileOutput(Script script) {
   @Override
   public String runScript(Script script, Map<String, Object> parameters) {
     String generatedScript = ScriptUtils.generateScript(script, parameters);
-    StringPythonOutputHandler handler = new StringPythonOutputHandler();
+    var handler = new ScriptOutputHandler();
```

Comment:
Too bad ScriptRunner interface is not streaming, it really should send progress messages and append to log. 🐛